cmd/argo/commands/cron/root.go

```diff
@@ -7,7 +7,7 @@ import (
 func NewCronWorkflowCommand() *cobra.Command {
 	var command = &cobra.Command{
 		Use:   "cron",
-		Short: "manage cron workflows\n\nNextScheduledRun assumes that the workflow-controller uses UTC as its timezone",
```

Comment:
I think you actually want to do this:

```golang
Short: 'Short: "manage cron workflows'
Long: `manage cron workflows

NextScheduledRun assumes that the workflow-controller uses UTC as its timezone
`
```

Reply:
Ah thanks. My first go PR.

Reply:
Copying this: https://github.com/argoproj/argo/blob/master/cmd/argo/commands/root.go#L27-L28

I haven't included the comment again in the Long field